transcribe.py

```diff
@@ -37,21 +41,29 @@ def transcribe_file(audio_path, tlog_path):
     except NotImplementedError:
         num_processes = 1
     with AudioFile(audio_path, as_path=True) as wav_path:
-        data_set = split_audio_file(wav_path,
-                                    batch_size=FLAGS.batch_size,
-                                    aggressiveness=FLAGS.vad_aggressiveness,
-                                    outlier_duration_ms=FLAGS.outlier_duration_ms,
-                                    outlier_batch_size=FLAGS.outlier_batch_size)
-        iterator = tf.data.Iterator.from_structure(data_set.output_types, data_set.output_shapes,
-                                                   output_classes=data_set.output_classes)
-        batch_time_start, batch_time_end, batch_x, batch_x_len = iterator.get_next()
-        no_dropout = [None] * 6
-        logits, _ = create_model(batch_x=batch_x, seq_length=batch_x_len, dropout=no_dropout)
-        transposed = tf.nn.softmax(tf.transpose(logits, [1, 0, 2]))
-        tf.train.get_or_create_global_step()
-        with tf.Session(config=Config.session_config) as session:
+        with tfv1.Session(config=Config.session_config) as session:
+            data_set = split_audio_file(wav_path,
+                                        batch_size=FLAGS.batch_size,
+                                        aggressiveness=FLAGS.vad_aggressiveness,
+                                        outlier_duration_ms=FLAGS.outlier_duration_ms,
+                                        outlier_batch_size=FLAGS.outlier_batch_size)
+
+            iterator = tfv1.data.Iterator.from_structure(tfv1.data.get_output_types(data_set),
+                                                         tfv1.data.get_output_shapes(data_set),
+                                                         output_classes=tfv1.data.get_output_classes(data_set))
+            batch_time_start, batch_time_end, batch_x, batch_x_len = iterator.get_next()
+
+            model = Model()
+            logits, _ = model(batch_x)
+            transposed = tf.nn.softmax(tf.transpose(logits, [1, 0, 2]))
```

Comment:
and here we still do `softmax(transpose)` ?

Reply:
Yeah, like I said above it's equivalent but I'll undo the other change.